alcs-frontend/src/app/features/planning-review/header/header.component.spec.ts

```diff
@@ -0,0 +1,53 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { RouterTestingModule } from '@angular/router/testing';
+
+import { HeaderComponent } from './header.component';
+
+describe('DetailsHeaderComponent', () => {
```

Comment:
Should this not match the name of the component 'HeaderComponent'?

Reply:
Yes it should, good catch. Fixed.